src/index.js

```diff
@@ -71,7 +71,7 @@ export default declare(({
       const rawSource = readFileSync(svgPath, 'utf8');
       const optimizedSource = state.opts.svgo === false
         ? { data: rawSource }
-        : optimize(rawSource, state.opts.svgo);
+        : optimize(rawSource, state.opts.svgo, svgPath);
```

Comment:
```suggestion
        : optimize(rawSource, { ...state.opts.svgo, path: svgPath });
```

Reply:
I didn't want to convert the options object to plain object in case 'state.opts.svgo' is falsy. It would cause some issues in optimize.js/validateAndFix.

Reply:
I've made the suggested change. thank you very much!